### PR TITLE
CS and doc fixes

### PR DESCRIPTION
**Pull request by GrahamCampbell**
1. Upgrade to `php-cs-fixer` 3.22.
2. Remove redundant `.php-cs-fixer` config and sort fixers by name.
3. Update phpdoc generator to remove double space characters.

### CODE REVIEW

File: src/CodeGenerator/src/Definition/Shape.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ private function __construct()
 
     /**
      * @param \Closure(string, Member|null=, array<string, mixed>=): Shape $shapeLocator
-     * @param \Closure(): ServiceDefinition $serviceLocator
+     * @param \Closure(): ServiceDefinition                                $serviceLocator
```
Comment (jderusse) [inline src/CodeGenerator/src/Definition/Shape.php:38]: Why this change

Comment (GrahamCampbell) [reply src/CodeGenerator/src/Definition/Shape.php:38]: php-cs-fixer made it - it is a bugfix with the alignment when there's a complex type

**Review by jderusse — APPROVED** [review]
